teachers_digital_platform/css/tdp.less

```diff
@@ -5,13 +5,28 @@
 // This is your project's main Less file.
 // Project-specific Less rules go after the imports.
 
+// Import Capital Frmework by reference so that variables and mixins are
+// available, but the whole output doesn't duplicate what we're already getting
+// from cfgov-refresh.
+@import (reference) 'cf-core.less';
+@import (reference) 'cf-icons.less';
+@import (reference) 'cf-buttons.less';
+@import (reference) 'cf-forms.less';
+@import (reference) 'cf-grid.less';
+@import (reference) 'cf-layout.less';
+@import (reference) 'cf-typography.less';
+@import (reference) 'cf-notifications.less';
+@import (reference) 'cf-pagination.less';
+@import (reference) 'cf-expandables.less';
+@import (reference) 'cf-tables.less';
```

Comment:
Good call on this `reference` Less feature. 👍

Reply:
We should probably do this on the bits of Owning a Home that haven't moved too.